<launch>
<include file="$(find finder_gazebo)/launch/finder_circuit_2.launch" />
<include file="$(find finder_bringup)/launch/finder_gmapping.launch" />
<include file="$(find exploration)/launch/Exploration_Services.launch" />
<node pkg="exploration" type="Prueba_Exploracion_Client.py" name="Exploration_and_Mapping" respawn="false" output="screen"></node>
</launch>
